src/routes/Receive.tsx

```diff
@@ -412,45 +406,82 @@ export default function Receive() {
                                 }}
                             >
                                 <MegaCheck />
-                                <FeeExplanation fee={lspFee()} />
+                                <LargeHeader centered>
+                                    {receiveState() === "paid" &&
+                                    paidState() === "lightning_paid"
+                                        ? "Payment Received"
+                                        : "Payment Initiated"}
+                                </LargeHeader>
                                 <Amount
-                                    amountSats={paymentInvoice()?.amount_sats}
+                                    amountSats={
+                                        receiveState() === "paid" &&
+                                        paidState() === "lightning_paid"
+                                            ? paymentInvoice()?.amount_sats
+                                            : paymentTx()?.received
```

Comment:
Would you prefer that the logic behind this be handled more similarly to `Send.tsx`. In other words the `sentDetails()?.amount` looks to see the transaction type ahead of time instead of using a ternary like it is here.